packages/vscode-extension/src/extension.ts

```diff
@@ -461,24 +453,3 @@ function migrateOldConfiguration() {
     Logger.error("Error when migrating old configuration", e);
   }
 }
-
-async function fixMacosBinary(context: ExtensionContext) {
```

Comment:
Out of curiosity, why is this no longer required?

Reply:
good point, forgot to mention this in the description, will do